File: C#/E3.AddFunctionality/S2.AddTest/CalculatorTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TDDLegacyCode.E3.AddFunctionality.S2.AddTest
{
    [TestClass]
    public class CalculatorTests : BaseCalculatorTests
    {
        [TestMethod]
        public void Number_after_minus_should_still_display()
        {
            Pressing("9-").ShouldShow("9");
        }
    }
}

File: src/testingil/webinar/tddlegacycode/e3/addfunctionality/s2/test/CalculatorTests.java
package testingil.webinar.tddlegacycode.e3.addfunctionality.s2.test;

import org.junit.jupiter.api.Test;

class CalculatorTests extends BaseCalculatorTests{

	@Test
	void number_after_minus_should_still_display() {
		pressing("9-").shouldShow("9");
	}

}
